solr_schema.xml

```diff
@@ -26,171 +26,278 @@ schema. In this case the version should be set to the next CKAN version number.
 (x.y but not x.y.z since it needs to be a float) -->
 <schema name="ckan" version="2.7">
 
-<types>
```

Comment:
Can you please split the update of the schema and the reformatting into two commits, so that it will be better visible what has changed?